### DISCUSSION TITLE
2023-06-27 CUE + Web Assembly Focus Session Call

### DISCUSSION THREAD

**Discussion by Lyoness** [cue-lang/cue#2463] [Announcements]
As announced on the [2026-06-20 community call](https://youtu.be/fH7DlV4hpho?t=1073), an initial version of Web Assembly support has landed in CUE! This discussion aims to gather feedback for the focus session on Tuesday 6/27 to hear more on we’re using CUE with Wasm, including future plans and how you can contribute. 

The session will be held **Tuesday, 27 June, 2023 at 1400 UTC**. 

Please reply with any questions/topics you would like to see covered.

### Meeting details

https://meet.google.com/xep-xsnj-kxw

This session will also be recorded for those unable to join us live.

### CUE Community shared calendar

[Subscribe to the shared calendar](https://calendar.google.com/calendar/u/0?cid=Y19lNzkxMWQ5OWQ4ZGIyMmU2ZTVjMzhkMTVkNjY2ZTVlNjdiNWE5ODNkZWU4N2JmNTU2NDY3NzI1OGIxYjJhMTFhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).  

### Recording

https://youtu.be/99gBJJiDHqU

### Slides/materials

https://docs.google.com/presentation/d/1PaAbydkiG8fp2mapIOYedw_UY1sHz7QG-T7-4tI9e9A/edit

### Auto-generated transcript 🆕 

To follow

**Comment by myitcv** [comment]
Updated the original description with links to the recording and slides.
